setup.py

```diff
@@ -15,7 +15,7 @@
     "click-log==0.1.8",
     "distro~=1.3",
     "dvc>=1.8.1",
-    "elasticsearch>=5.0",
+    "elasticsearch>=5.0,<7.14",
```

Comment:
Why do we have elasticsearch both here and in extras?

Reply:
Also, can we add a comment here on why we're limiting the version?  Something like `# elasticsearch-py 7.14 breaks backwards compatibility with servers prior to 7.11`